invenio_rdm_records/mappings/v6/records/record-v1.0.0.json

```diff
@@ -81,11 +83,100 @@
             }
           }
         },
+        "dates": {
+          "type": "object",
+          "properties": {
+            "start": {
+              "type": "date"
+            },
+            "end": {
+              "type": "date"
+            },
+            "type": {
+              "type": "keyword"
+            },
+            "description": {
+              "type": "text"
+            }
+          }
+        },
+        "description": {
+          "type": "text"
+        },
+        "embargo_date": {
+          "type": "date"
+        },
+        "keywords": {
+          "type": "keyword"
+        },
+        "language": {
+          "type": "keyword"
+        },
+        "owners": {
+          "type": "integer"
+        },
+        "publication_date": {
+          "type": "date"
+        },
+        "recid": {
+          "type": "keyword"
+        },
+        "resource_type": {
+          "type": "object",
+          "properties": {
+            "openaire_subtype": {
+              "type": "keyword"
+            },
+            "type": {
+              "type": "keyword"
+            },
+            "subtype": {
+              "type": "keyword"
+            }
+          }
+        },
+        "rights": {
+          "type": "object",
+          "properties": {
+            "rights": {
+              "type": "text"
+            },
+            "uri": {
+              "type": "keyword"
+            },
+            "identifier": {
+              "type": "keyword"
+            },
+            "identifier_scheme": {
+              "type": "keyword"
+            },
+            "scheme_uri": {
+              "type": "keyword"
+            },
+            "lang": {
+              "type": "keyword"
+            }
+          }
+        },
+        "suggest_title": {
+          "type": "completion"
+        },
+        "title": {
+          "type": "text",
+          "copy_to": "suggest_title"
+        },
+        "version": {
+          "type": "keyword"
+        },
         "_created": {
           "type": "date"
         },
         "_updated": {
           "type": "date"
```

Comment:
These were deleted from the jsonschema and should be deleted here too, right?

Reply:
Those are *system/operational* metadata, that does is not sent by the user (when creating a record). Therefore, is not in the JSONSchema that gets validated). However, it is added and sent to Elasticsearch. 

See Zenodo for reference:

- JSONschema: https://github.com/zenodo/zenodo/blob/master/zenodo/modules/records/jsonschemas/records/record-v1.0.0.json
- ES7 mapping: https://github.com/zenodo/zenodo/blob/master/zenodo/modules/records/mappings/v7/records/record-v1.0.0.json#L634